node/src/sub_lib/accountant.rs

```diff
@@ -37,11 +38,6 @@ lazy_static! {
         threshold_interval_sec: 21600,
         unban_below_gwei: 500_000_000,
     };
-    pub static ref DEFAULT_SCAN_INTERVALS: ScanIntervals = ScanIntervals {
```

Comment:
This was better, I don't know why you switched. No need to make changes, it'll be a waste of time.